www/less/widenImages.less

```diff
@@ -4,31 +4,9 @@
     display: block;
     margin-left: auto;
     margin-right: auto;
-}
-
-@media (orientation: portrait) {
-    .wideImageOverride {
-        width:100% !important;
-        max-width:100% !important;
-        height:auto !important;
-    }
-
-    /* Tablet override */
-    @media (min-device-width: 768px) {
-        .wideImageOverride {
-            width:60% !important;
```

Comment:
That would leave in place the landscape narrowing and prevent any of the widening JS logic from breaking.